target/linux/stm32/base-files/lib/preinit/82_brcmfmac_fix_fw_names

```diff
@@ -0,0 +1,15 @@
+
+. /lib/functions/system.sh
+
+brcmfmac_fix_fw_names() {
+	local fw="/lib/firmware/brcm/brcmfmac43430-sdio"
+	local board="$(board_name)"
+
+	for ext in bin clm_blob txt
+	do
+		[ ! -f "${fw}.${board}.${ext}" ] && [ -f "${fw}.${ext}" ] && \
+			ln -s "${fw##*/}.${ext}" "${fw}.${board}.${ext}"
```

Comment:
Why is this needed?
Why don't you directly create the symlinks at build time?

Reply:
I think showing the error message is fine if it falls back to the generic file for which you created a symlink.

Reply:
Ok, so I remove it